<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>831</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="mainBoard">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>600</width>
      <height>600</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="NameLabel">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>20</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Name:</string>
    </property>
   </widget>
   <widget class="QLabel" name="AttackLabel">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>60</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Attack: </string>
    </property>
   </widget>
   <widget class="QLabel" name="HealthLabel">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>40</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Health: </string>
    </property>
   </widget>
   <widget class="QLabel" name="DefenseLabel">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>80</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Defense: </string>
    </property>
   </widget>
   <widget class="QLabel" name="SpeedLabel">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>100</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Speed: </string>
    </property>
   </widget>
   <widget class="QLabel" name="PlayerLabel">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>120</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Player: </string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>0</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Player</string>
    </property>
   </widget>
   <widget class="QPushButton" name="itemButton">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>170</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Use Item</string>
    </property>
   </widget>
   <widget class="QLabel" name="itemLabel">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>136</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Item:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>230</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Opponent</string>
    </property>
   </widget>
   <widget class="QLabel" name="SpeedLabel_2">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>330</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Speed:</string>
    </property>
   </widget>
   <widget class="QLabel" name="PlayerLabel_2">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>350</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Player:</string>
    </property>
   </widget>
   <widget class="QLabel" name="HealthLabel_2">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>270</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Health:</string>
    </property>
   </widget>
   <widget class="QLabel" name="DefenseLabel_2">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>310</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Defense:</string>
    </property>
   </widget>
   <widget class="QLabel" name="NameLabel_2">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>250</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Name:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>370</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Item:</string>
    </property>
   </widget>
   <widget class="QLabel" name="AttackLabel_2">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>290</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Attack:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="attackButton">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>400</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Attack</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="gameStats">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>620</y>
      <width>231</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="gameUpdates">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>620</y>
      <width>541</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="turnButton">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>530</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Advance Turn</string>
    </property>
   </widget>
   <widget class="QPushButton" name="movingHold">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>450</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Show who can move (hold)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="attackingHold">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>490</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Show who can attack (hold)</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>580</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="duelButton">
       <property name="text">
        <string>New Duel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="gameButton">
       <property name="text">
        <string>New Game</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
